phoenix /var/tmp/portage/dev-python/chromadb-1.4.1/work # diff -uPNr chromadb-1.4.1 chromadb-1.4.1.new/
diff '--color=auto' -uPNr chromadb-1.4.1/Cargo.toml chromadb-1.4.1.new/Cargo.toml
--- chromadb-1.4.1/Cargo.toml   2006-07-24 03:21:28.000000000 +0200
+++ chromadb-1.4.1.new/Cargo.toml       2026-01-18 18:24:35.873420099 +0100
@@ -67,7 +67,8 @@
 rayon = "1.10.0"
 validator = { version = "0.19", features = ["derive"] }
 rust-embed = { version = "8.5.0", features = ["include-exclude", "debug-embed"] }
-hnswlib = { version = "0.8.2", git = "https://github.com/chroma-core/hnswlib.git", branch = "master" }
+hnswlib = { path = "vendor/hnswlib" }
+
 reqwest = { version = "0.12.9", features = ["rustls-tls-native-roots", "http2"], default-features = false }
 random-port = "0.1.1"
 ndarray = { version = "0.16.1", features = ["approx"] }
